### PR TITLE
feat: Added NotImplemented Attribute for Page.TopAppBar and Page.BottomAppBar

### PR DESCRIPTION
**Pull request by ahoefling**
GitHub Issue (If applicable): #1514

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
The `Page.TopAppBar` and `Page.BottomAppBar` is not implemented and only works on UWP

## What is the new behavior?
Added the `[Uno.NotImplemented]` attribute over the `Page.TopAppBar` and `Page.BottomAppBar` to clearly communicate that the feature is not implemented in 
* macOS
* Android
* iOS
* WASM

Adding the attribute is following the recommendation in issue #1514

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by ahoefling** [comment]
I linked the wrong issue when creating the PR. I just updated it and this PR fixes #1514.

File: src/Uno.UI/UI/Xaml/Controls/Page/Page.cs [hunk 1]
```diff
@@ -19,13 +19,15 @@ public Page()
 		protected internal virtual void OnNavigatingFrom(NavigatingCancelEventArgs e) { }
 
 		#region TopAppBar
-
+#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/Page/Page.cs:22]:
```suggestion
```

File: src/Uno.UI/UI/Xaml/Controls/Page/Page.cs [hunk 2]
```diff
@@ -36,17 +38,19 @@ public AppBar TopAppBar
 					FrameworkPropertyMetadataOptions.ValueInheritsDataContext
 				)
 			);
-
+#endif
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/Page/Page.cs:41]:
```suggestion
```

File: src/Uno.UI/UI/Xaml/Controls/Page/Page.cs [hunk 3]
```diff
@@ -36,17 +38,19 @@ public AppBar TopAppBar
...
 		#endregion
 
 		#region BottomAppBar
-
+#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/Page/Page.cs:45]:
```suggestion
```

File: src/Uno.UI/UI/Xaml/Controls/Page/Page.cs [hunk 4]
```diff
@@ -57,7 +61,7 @@ public AppBar BottomAppBar
 					FrameworkPropertyMetadataOptions.ValueInheritsDataContext
 				)
 			);
-
+#endif
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/Page/Page.cs:64]:
```suggestion
```

**Comment by ahoefling** [comment]
Thanks for the feedback, I wasn't sure if it was needed or not. I have gone ahead and removed the conditional compilation.

**Pull request merged**